HW0/P5/P5.txt

```diff
@@ -1 +1,327 @@
-# Your answers here.
+###############################
+
+# CS 205 Fall 2015 Homework 0 Problem 5
+# Submitted by Kendrick Lo (Harvard ID: 70984997)
+# Github username: ppgmg
+
+Summary
+-------
+
+1. 255 seconds
+2. 34 seconds
+3. 8 seconds
+4. log N seconds
+5. 15 seconds
+6. 272 seconds, 143 seconds
+   Wait-and-Combine (16 workers): 39 seconds
```

Comment:
2 bags: Same idea as problem 5, for the handing time, from 256—>1 which is cost 8 seconds. Therefore, the total time will be 15 + 8 = 23 seconds.